presto-hive/src/main/java/io/prestosql/plugin/hive/rcfile/RcFilePageSourceFactory.java

```diff
@@ -188,7 +188,8 @@ public static TextRcFileEncoding createTextVectorEncoding(Properties schema, Dat
 
         // the first three separators are set by old-old properties
         separators[0] = getByte(schema.getProperty(FIELD_DELIM, schema.getProperty(SERIALIZATION_FORMAT)), DEFAULT_SEPARATORS[0]);
-        separators[1] = getByte(schema.getProperty(COLLECTION_DELIM), DEFAULT_SEPARATORS[1]);
+        // for map field collection deliminator, hive 1.x use "collection.delim" but hive 3.x use "colelction.delim" 
+        separators[1] = getByte(schema.getProperty(COLLECTION_DELIM, schema.getProperty("colelction.delim")), DEFAULT_SEPARATORS[1]);
```

Comment:
can you link the issue in HIVE as comment here

Reply:
This comment seems to be reversed (new Hive versions use the correct name without a typo)

Reply:
@kokosing done,  thanks for advising.